src/original-content/VirtualFileSystem.cpp

```diff
@@ -1,6 +1,6 @@
 #include "VirtualFileSystem.hpp"
-#include <Error/BsException.h>
 #include <FileSystem/BsFileSystem.h>
+#include <excepction/Throw.hpp>
```

Comment:
We should fix that typo soon, before everything get's too big. But that's something for another PR.

Reply:
Oh, that absolutely went past me. We should definitely fix this!

Reply:
I can do a PR for that. It's currently only 13 files which have to be fixed (+ the CMakeLists.txt where I forgot to include it in my PR #4).